### PR TITLE
[Metricbeat][Docker] Docker cpu norms

### PR DESCRIPTION
**Pull request by ChrsMark**
This PR implements the proposal for `docker.cpu.*.norm.pct` fields described on https://github.com/elastic/beats/issues/13343.

### CODE REVIEW

**Comment by ChrsMark** [comment]
> Hey, can we get Godocs for the rest of the exported methods in `helper.go` ? Surprised hound didn't complain.

👍 sure! thank you!

**Comment by jsoriano** [comment]
Also a changelog entry will be needed.

File: metricbeat/module/docker/cpu/cpu_test.go [hunk 1]
```diff
@@ -34,42 +34,43 @@ func cpuUsageFor(stats types.StatsJSON) *cpuUsage {
 		Stat:        &docker.Stat{Stats: stats},
 		systemDelta: 1000000000, // Nanoseconds in a second
 	}
-	if len(stats.CPUStats.CPUUsage.PercpuUsage) == 0 {
-		u.cpus = 1
-	}
+	//if len(stats.CPUStats.CPUUsage.PercpuUsage) == 0 {
+	//	u.cpus = 1
+	//}
```
[the remark this answers is not in the document]
Comment (ChrsMark) [reply metricbeat/module/docker/cpu/cpu_test.go]: 🙈 thank you for catching this!

**Comment by ChrsMark** [comment]
> In principle apart of the small comment it LGTM.
> 

Thank you for reviewing this Jaime!

> I have a couple of questions before merging:
> 
> In the system modules, the cpu metrics collected are optional and managed by the [`cpu.metrics` setting](https://www.elastic.co/guide/en/beats/metricbeat/7.3/metricbeat-metricset-system-cpu.html#_configuration_3). It can be useful if more methods are added, though I think that not really needed because the overhead doesn't look so big. Did you consider doing something similar here?
> 
> Not to be done in this PR, but I think that it could be good to expose the data to calculate percentages in the events, so custom percentages can be calculated on query time. Now we have something like the number of cpus of the host in `docker.cpu.system.pct`, but it is a bit counterintuitive, should we add a explicit field for that? like `docker.cpu.online_cpus`?

Yeap I was thinking about this too, however I didn't add this yet since I was not sure enough because it was not already there for `pct` and `ticks`. 
I'm really interested in having this extension in this metriset too and I could work on this. We could open another issue to discuss the specs, WDYT?

**Pull request merged**